internal/receiver/smartagentreceiver/receiver.go

```diff
@@ -175,6 +176,9 @@ func (r *Receiver) createMonitor(monitorType string, host component.Host) (monit
 		output.AddExtraDimension(k, v)
 	}
 
+	systemTypeVal := fmt.Sprintf("smartagent-%s", monitorType)
```

Comment:
I don't think that we should prefix the type with _**smartagent-**_ the type should be _**kafka**_ for example, when we move to the OTel receivers the type should be the same there.

Reply:
Many of the monitor types start with **collectd** or **telegraf** like **_collectd/elasticsearch_** , we need to make sure that collectd/telegraf is not part of the system.type (took the type from https://docs.signalfx.com/en/latest/integrations/agent/monitors/index.html)

Reply:
Ah, ok. Will fix.

Reply:
I am not sure if that is indeed the case with the monitorType variable you are using

Reply:
I ran a test and the `monitorType` variable does have the `collectd/` prefix. Should we strip all `foo/` prefixes or just `collectd/` and `telegraf/`? Stripping all would strip `prometheus/` as well. There would be a little bit of overlap because we have a `prometheus/postgres` monitor as well as a `postgresql` one, but we'd still be able to tell the difference because one is `postgres` and the other is `postgresql`.

Reply:
I think that stripping all makes sense, collectd/telegraf/prometheus are not part of the type.